Packs/qualys/Playbooks/playbook-Launch_And_Fetch_Scan_Based_Findings_Report_-_Qualys.yml

```diff
@@ -155,6 +155,45 @@ tasks:
       exitCondition: ""
       wait: 1
       max: 0
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 86f0e348-c8e9-4d57-845b-e77ccabb919d
+    type: regular
+    task:
+      id: 86f0e348-c8e9-4d57-845b-e77ccabb919d
+      version: -1
+      name: Try to get the created scan metadata
```

Comment:
Please change to `Get the created scan metadata`.
Also, we said - you need to add a conditional task after this task to verify that you got the scan ID. if you didnt (after the retries) you will NOT enter the polling subplaybook.